subprojects/language-native/src/main/java/org/gradle/language/assembler/tasks/Assemble.java

```diff
@@ -102,6 +104,7 @@ public void assemble() {
         setDidWork(result.getDidWork());
     }
 
+    @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Is it `RELATIVE` or `NAME_ONLY` or `IGNORE`?

Reply:
Includes _are_ relative. But do the compiled object files reflect the source hierarchy? If not, then I think this is just `NAME_ONLY`.